shell/demo/demo3.ps1

```diff
@@ -0,0 +1,173 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true, HelpMessage="Prelude Token")]
+    [String]$preludeToken
+)
+
+function FromEnv { param ([string]$envVar, [string]$default)
+    $envVal = [Environment]::GetEnvironmentVariable($envVar, "User")
+    if ($envVal) { return $envVal } else { return $default }
+}
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="07d31938-92fa-40aa-b8af-77a54055f481"
+$TEST_INTRO="Will your computer quarantine a malicious executable file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Windows executable was used by the SteelClover group 
+during an attack in January 2023.
+
+This test will attempt to download a malicious file to your disk - a defanged version of RedLine Stealer used by SteelClover - 
+to see how your machine will respond. Downloading malicious files can open you up to further attacks,
```

Comment:
do we want to say 'how your machine will respond' or 'how your endpoint defenses will respond'?

Reply:
ah yes let I'll make that change.